service_status/lambda_handler.py

```diff
@@ -0,0 +1,72 @@
+import json
+import re
+import traceback
+
+from common.constant import NETWORKS
+from common.repository import Repository
+from common.utils import Utils
+from service_status.service_status import ServiceStatus
+
+NETWORKS_NAME = dict((NETWORKS[netId]['name'], netId) for netId in NETWORKS.keys())
+obj_util = Utils()
+db = dict((netId, Repository(net_id=netId)) for netId in NETWORKS.keys())
+route_path = {}
+
+
+def request_handler(event, context):
+    print(event)
+    if 'path' not in event:
+        print("request_handler::path: ", None)
+        return get_response("400", "Bad Request")
+    try:
+        payload_dict = None
+        path = event['path'].lower()
+        if event['httpMethod'] == 'POST':
+            body = event['body']
+            if body is not None and len(body) > 0:
+                payload_dict = json.loads(body)
+                print("Processing [" + str(path) + "] with body [" + str(body) + "]")
+        elif event['httpMethod'] == 'GET':
+            payload_dict = event.get('queryStringParameters')
+            print("Processing [" + str(path) + "] with queryStringParameters [" + str(payload_dict) + "]")
+        stage = event['requestContext']['stage']
+        net_id = NETWORKS_NAME[stage]
```

Comment:
Lets initialize the DB connection outside the handler to allow for reuse across calls

Reply:
already taken care. Now I will raise an exception if the DB connection is not initialized.